src/generate-require-signin-wrapper.tsx

```diff
@@ -12,29 +12,40 @@ const generateRequireSignInWrapper = (
 ): RequireSignInWrapper => {
   const requireSignInWrapper = (PageComponent: ComponentClass): ComponentClass => {
     interface WrapperProps {
+      readonly hasVerificationBeenAttempted: boolean
       readonly isSignedIn: boolean
       readonly history: {
         readonly replace: (path: string) => void
       }
     }
 
     class GatedPage extends React.Component<WrapperProps> {
-      public componentWillMount (): void {
+      public componentWillReceiveProps(nextProps: any): void {
```

Comment:
can we type `nextProps` with `WrapperProps`? 😃

Reply:
Ah yes, forgot to do that. Just pushed adc99c6